manifests/limits.pp

```diff
@@ -33,6 +36,7 @@
   Stdlib::Absolutepath $limits_d_dir   = '/etc/security/limits.d',
   Stdlib::Filemode $limits_d_dir_mode  = '0750',
   Boolean $purge_limits_d_dir          = false,
+  Optional[Variant[String[1], Array]] $purge_limits_d_dir_ignore = undef,
```

Comment:
Perhaps this should be array of non empty strings instead of just an array.

Reply:
This has been updated to require an array of non-empty strings.